addons/binding/org.openhab.binding.wifiled/src/main/java/org/openhab/binding/wifiled/handler/FadingWiFiLEDDriver.java

```diff
@@ -12,41 +12,56 @@
 import org.eclipse.smarthome.core.library.types.OnOffType;
 import org.eclipse.smarthome.core.library.types.PercentType;
 import org.eclipse.smarthome.core.library.types.StringType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.io.DataOutputStream;
 import java.io.IOException;
 import java.net.NoRouteToHostException;
 import java.net.Socket;
 import java.net.SocketException;
 import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
 import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.Semaphore;
 import java.util.concurrent.TimeUnit;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+import java.util.function.Function;
 
 /**
  * The {@link FadingWiFiLEDDriver} class is responsible for the communication with the WiFi LED controller.
  * It utilizes color fading when changing colors or turning the light on of off.
  *
  * @author Stefan Endrullis
+ * @author Ries van Twisk
  */
 public class FadingWiFiLEDDriver extends AbstractWiFiLEDDriver {
 
     public static final int DEFAULT_FADE_DURATION_IN_MS = 1000;
     public static final int DEFAULT_FADE_STEPS = 100;
+    public static final int KEEPCOMMOPENINMS = 1000; // After last fade keep communication channel open fs XXX MS
```

Comment:
Normally you would use _ to separate different words in a constant: `KEEP_COMMUNICATION_OPEN_FOR_MS `